ember-flight-icons/tests/dummy/app/components/ds-footer.hbs

```diff
@@ -0,0 +1,17 @@
+<footer class="ds-footer">
+    <div class="flex sm:justify-center md:justify-start w-full md:w-1/2 md:h-full">
+      <p>
+        &copy; 2021 HashiCorp, Inc. All rights reserved.
+      </p>
+    </div>
+    <div class="flex justify-evenly w-full md:w-1/2 md:h-full">
+      <p><b>Contact Us</b></p>
+      <a href="https://hashicorp.slack.com/archives/C7KTUHNUS" class="ds-a">
+        #team-design-systems (Slack)
+      </a>
+
+      <a href="mailto:design-systems@hashicorp.com" class="ds-a">
+        design-systems@hashicorp.com
```

Comment:
I think it'd be better not to publicize this specific email address externally as we don't necessarily want inbound email from outside the company

Reply:
Should we also not link to internal slack then? 

The point of adding contact info in the footer is that it's an accessibility conformance criteria, to provide a way for users to get in contact with us in some way.